test/relations.test.js

```diff
@@ -2776,10 +2953,14 @@ describe('relations', function() {
     it('should find record on scope with promises', function(done) {
       Passport.findOne()
       .then(function(p) {
-        p.personId.should.eql(personCreated.id);
+        if (!isCassandraConnector) {
+          // Not sorted on Cassandra; so, may or may not be the same
+          p.personId.should.eql(personCreated.id);
+        }
         return p.person.getAsync();
       })
       .then(function(person) {
+        if (person === undefined && isCassandraConnector) return done();
```

Comment:
This is evil - the verification is completely skipped yet the test is marked as passed. Let's fix the test to always use the same Person instance, as I commented above.